### PR TITLE
Api connect

### PR DESCRIPTION
**Pull request by Sajeelzafar**
### In this pull request, I have performed the following tasks:

- Use fetch to add and remove books with createAsyncThunk from the APIs.
- Print book list using the API on DOM.
- Delete the book from DOM using the API.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+//     },
+//     body: JSON.stringify({
+//       item_id: "item1",
+//       title: "The Great Gatsby",
+//       author: "John Smith",
+//       category: "Fiction"
+//     }),
+//   }).then((resp) => resp.text());
+//   console.log(ff());
+
```
Comment (Mihndim2020) [inline src/components/Book.js]: - [x] It will be nice to remove commented code from you code base as this makes your code dirty and less readable.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Sajeelzafar, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Sajeelzafar,

Good job on removing the commented code on your codebase as requested by the previous reviewer. 👍 
Your project is complete!  

## Highlights:
✅ You used Gitflow correctly
✅ Linter checks passed
✅ Add and remove features works fine
✅ Used Redux well 👍 

## Recommendation:

- [x] How about you remove the prefix texts on a book and an author? Basically, from the template design, it's not there and it's redundant to tell. I underlined the text in the screenshot.
- [x] Also, you could clear the input fields after a book is submitted 👍 

image

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Sajeelzafar** [comment]
Thank you for the time to review my project @AmaduKamara. Yes, surely those recommendations are ones I am looking forward to implementing in my next project. I'll be adding styling and making my design similar to the given template.

**Pull request merged**